packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -117,9 +121,11 @@ describe('ReactDOMInput', () => {
     );
   });
 
-  it('should warn for controlled value of "" with missing onChange', () => {
-    expect(() => {
-      ReactDOM.render(<input type="text" value="" />, container);
+  it('should warn for controlled value of "" with missing onChange', async () => {
+    await expect(async () => {
+      await act(() => {
```

Comment:
fwiw, i've been using `flushSync` for error assertions, because if a warning or other error is thrown this will crash jest watch mode